assembly/src/assembler/linker/rewrites/module.rs

```diff
@@ -68,8 +68,13 @@ impl<'a, 'b: 'a> ModuleRewriter<'a, 'b> {
             kind,
         };
         match self.resolver.resolve_target(&caller, target) {
-            Err(err) => return ControlFlow::Break(err),
-            Ok(ResolvedTarget::Phantom(_)) => (),
+            Err(err) => {
+                log::error!(target: "linker", "    | failed to resolve target {target}");
+                return ControlFlow::Break(err);
+            },
+            Ok(ResolvedTarget::Phantom(_)) => {
+                log::warn!(target: "linker", "    | resolved phantom target {target}");
```

Comment:
Question: does "phantom" now mean "dynamic"?

Reply:
Yes and no - a phantom reference is inherently dynamically-linked, because it refers to a MAST root we don't have any information for. However, the reason we call this a phantom reference and not a dynamic one, is because we want to be clear that the reference is to something unknown to us, a definition we can't see.

We could call it `Dynamic`, but that wouldn't fully capture the semantics of what `Phantom` means in the linker.